Gist/Managers/MessageQueueManager.swift

```diff
@@ -18,6 +19,26 @@ class MessageQueueManager {
         }
     }
 
+    func checkLocalStoreForMessages() {
```

Comment:
Would this function benefit from a rename [such as in Android](https://github.com/customerio/gist-android/pull/2#discussion_r1098502444)?

Reply:
Updated both this and another internal method to match.